tests/github-action-helper.sh

```diff
@@ -104,6 +104,38 @@ wait_for_operator_pod_to_be_ready_state_custom() {
 EOF
 }
 
+wait_for_pod_running() {
```

Comment:
Since we wait for either running or deleted, how about a more generic name again :) 
```suggestion
wait_for_pod() {
```

Reply:
Since #61 is merged now, how about updating to call `wait_for_deployment_to_be_running`? Then I think this PR will be ready for merge.